### PR TITLE
feat: add relationships to other docs section

### LINKED ISSUE #128: What is the benefit of defining a VC-like data model with JWT claims?
Data model in this document resembles to the ID Token data model as defined in OIDC (with the addition of cnf).

On the other hand, the data model is trying to follow the W3C VC data model.
What is the benefit of this data model over the W3C VC data model (except that the claim names are different and iss can be a URI, and not an object and that JSON-LD is not supported by the data model, which basically excludes some significant use cases such as Europass that are leveraging JSON-LD for internationalization, for example)?

### PR DESCRIPTION
**Pull request by awoie**
This PR:
- adds relationship to other docs section
- fixes #128 
- preview here: https://drafts.oauth.net/oauth-sd-jwt-vc/awoie.fix-128/draft-ietf-oauth-sd-jwt-vc.html

### CODE REVIEW

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 1]
```diff
@@ -568,7 +568,13 @@ in the Issuer identifier.
 
 # Relationships to Other Documents
 
-TBD
+This specification defines validation and processing rules for verifiable credentials using a JSON-only
+payloads and secured by SD-JWT [@!I-D.ietf-oauth-selective-disclosure-jwt]. Other specifications exist that aim
+to have similar scope. For example, W3C Verifiable Credential Data Model (VCDM) 2.0 [@W3C.VCDM] defines
```
Comment (danielfett) [inline draft-ietf-oauth-sd-jwt-vc.md:573]: 'similar scope' sounds as if those other documents also use JSON-only payloads with SD-JWT. I think they can be compared in that they all define credential formats.

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 2]
```diff
@@ -568,7 +568,13 @@ in the Issuer identifier.
 
 # Relationships to Other Documents
 
-TBD
+This specification defines validation and processing rules for verifiable credentials using a JSON-only
```
Comment (danielfett) [inline draft-ietf-oauth-sd-jwt-vc.md:571]:
```suggestion
This specification defines validation and processing rules for verifiable credentials using JSON-only
```

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 3]
```diff
@@ -568,7 +568,13 @@ in the Issuer identifier.
...
+payloads and secured by SD-JWT [@!I-D.ietf-oauth-selective-disclosure-jwt]. Other specifications exist that aim
+to have similar scope. For example, W3C Verifiable Credential Data Model (VCDM) 2.0 [@W3C.VCDM] defines
+a data model for verifiable credentials encoded as JSON-LD, ISO/IEC 18013-5:2021 [@ISO.18013-5] defines a representation
+of verifiable credentials in the mobile document (mdoc) format encoded as CBOR and secured using COSE. Each
+specification can be independently used while a mapping between different data model representations might be possible.
```
Comment (danielfett) [inline draft-ietf-oauth-sd-jwt-vc.md:576]: As discussed on our call, I'm not sure how this helps the reader. Theoretically I can map between all tree structures. But what could be the purpose of that excercise and why do we need to mention it here?

File: draft-ietf-oauth-sd-jwt-vc.md (hunk 1 above)
Comment (awoie) [reply draft-ietf-oauth-sd-jwt-vc.md]: I updated the text.

File: draft-ietf-oauth-sd-jwt-vc.md (hunk 3 above)
Comment (awoie) [reply draft-ietf-oauth-sd-jwt-vc.md]: I updated the text.

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 4]
```diff
@@ -568,7 +568,12 @@ in the Issuer identifier.
 
 # Relationships to Other Documents
 
-TBD
+This specification defines validation and processing rules for verifiable credentials using JSON-only
+payloads and secured by SD-JWT [@!I-D.ietf-oauth-selective-disclosure-jwt]. Other specifications exist
+that define their own verifiable credential formats, which can be used independently of each other. For example, W3C Verifiable
+Credential Data Model (VCDM) 2.0 [@W3C.VCDM] defines a data model for verifiable credentials encoded as JSON-LD,
+ISO/IEC 18013-5:2021 [@ISO.18013-5] defines a representation of verifiable credentials in the mobile document (mdoc)
```
Comment (bc-pi) [inline draft-ietf-oauth-sd-jwt-vc.md:575]:
```suggestion
Credential Data Model (VCDM) 2.0 [@W3C.VCDM] defines a data model for verifiable credentials encoded as JSON-LD and
ISO/IEC 18013-5:2021 [@ISO.18013-5] defines a representation of verifiable credentials in the mobile document (mdoc)
```

**Comment by bc-pi** [comment]
One little nit on the text but otherwise looks okay (after the couple iterations by @awoie and @danielfett, thanks for those).

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 5]
```diff
@@ -568,7 +568,12 @@ in the Issuer identifier.
 
 # Relationships to Other Documents
 
-TBD
+This specification defines validation and processing rules for verifiable credentials using JSON-only
```
Comment (Sakurann) [inline draft-ietf-oauth-sd-jwt-vc.md:571]: `using JSON-only` stood out here for me. `using only JSON payload` or just `using JSON payloads` should be fine? (sure, some can say JSON-LD is also JSON, but JSON-only does not really make that point clearer IMO)

Comment (awoie) [reply draft-ietf-oauth-sd-jwt-vc.md:571]: We could also add something like:
> Compared to other credential formats, SD-JWT VC was created to be focused on developers who use JSON-based data models and who are familiar with the JOSE stack.

Comment (bc-pi) [reply draft-ietf-oauth-sd-jwt-vc.md:571]: @Sakurann's suggestion of just `using JSON payloads` seems good 
```suggestion
This specification defines validation and processing rules for verifiable credentials using JSON
```

File: draft-ietf-oauth-sd-jwt-vc.md [hunk 6]
```diff
@@ -568,7 +568,12 @@ in the Issuer identifier.
 
 # Relationships to Other Documents
 
-TBD
+This specification defines validation and processing rules for verifiable credentials using JSON
+payloads and secured by SD-JWT [@!I-D.ietf-oauth-selective-disclosure-jwt]. Other specifications exist
+that define their own verifiable credential formats, which can be used independently of each other. For example, W3C Verifiable
```
Comment (danielfett) [inline draft-ietf-oauth-sd-jwt-vc.md:573]:
```suggestion
that define their own verifiable credential formats; for example, W3C Verifiable
```

**Review by danielfett — APPROVED** [review]
Approved with a minor proposed improvment.

**Comment by awoie** [comment]
@Sakurann pls approve if you are happy with the latest language, then we will merge the PR.

**Review by Sakurann — APPROVED** [review]

**Pull request merged**